core-plugin/README.adoc

```diff
@@ -13,3 +13,60 @@ This plugin offers the following functionality:
 === Compatibility
 
 This plugin requires link:http://gradle.org[Gradle] 4.3 or better
+
+=== Installing
+
+.build.gradle
+[source, groovy]
+----
+plugins {
+  id 'com.github.jruby-gradle.core' version 'VERSION-OF-PLUGIN'
+}
+----
+
+=== Adding repositories
+
+.build.gradle
+[source,groovy]
+----
+repositories {
+    ruby.mavengems() // <1>
+    ruby.mavengems('https://foo.bar') // <2>
+    ruby.mavengems('acme-rubygems', 'https://foo.bar') // <3>
+
+    ruby.gems() // <4>
+    ruby.gems { // <5>
+        prerelease = true
+    }
+    ruby.gems('https://foo.bar') // <6>
+    ruby.gems('acme-rubygems', 'https://foo.bar') // <7>
+}
+----
+<1> Adds a Maven repository that uses the one official supported by the JRuby group. In order to use this, GEM dependencies should all be placed in the `rubygems` group.
```

Comment:
My suggestion would be list this after `ruby.gems()`. Right now the repository which @JRuby points to is just a CNAME to rubygems-proxy.torquebox.org, which they convinced the Torquebox people to turn back on.

I think we should be encouraging people to rely on the built-in proxying you created

Reply:
Cool. I'll make the change.